src/internal/m365/graph/errors.go

```diff
@@ -62,16 +62,18 @@ const (
 type errorMessage string
 
 const (
-	IOErrDuringRead   errorMessage = "IO error during request payload read"
-	MysiteURLNotFound errorMessage = "unable to retrieve user's mysite url"
-	MysiteNotFound    errorMessage = "user's mysite not found"
-	NoSPLicense       errorMessage = "Tenant does not have a SPO license"
+	IOErrDuringRead         errorMessage = "IO error during request payload read"
+	MysiteURLNotFound       errorMessage = "unable to retrieve user's mysite url"
+	MysiteNotFound          errorMessage = "user's mysite not found"
+	NoSPLicense             errorMessage = "Tenant does not have a SPO license"
+	UnableProvisionResource errorMessage = "unable to provision resource" // happens if group sharepoint site not available
 )
 
 const (
 	LabelsMalware             = "malware_detected"
 	LabelsMysiteNotFound      = "mysite_not_found"
 	LabelsNoSharePointLicense = "no_sharepoint_license"
+	LabelsNoProvisionResource = "no_provision_resource"
```

Comment:
nit: Wording here is a bit clunky.
```suggestion
	LabelsCannotProvisionResource = "cannot_provision_resource"
```